### ISSUE TITLE
Patch credits changed in 3.12.0 branch

### ISSUE THREAD

**Issue by isuruf** [AnacondaRecipes/python-feedstock#121]
In https://github.com/AnacondaRecipes/python-feedstock/pull/113 I see that patch credits have been changed.
For eg:

```diff
-From e8a6dc5a388a6cc18d15ce3485d698ad0f0b187a Mon Sep 17 00:00:00 2001
-From: Isuru Fernando <isuruf@gmail.com>
-Date: Wed, 7 Oct 2020 10:08:30 -0500
-Subject: [PATCH 18/25] Fix TZPATH on windows
+From 5d81879c38e7c3f10e4c63636f17cc36fb170980 Mon Sep 17 00:00:00 2001
+From: Charles Bousseau <cbousseau@anaconda.com>
+Date: Thu, 25 May 2023 17:01:32 -0400
+Subject: [PATCH] Fix TZPATH on windows
```

Can you keep the From field intact or add me using a `Co-Authored-By` tag?

cc @jezdez, @chenghlee

**Comment by isuruf** [comment]
cc @cbouss

**Comment by cbouss** [comment]
Sorry about that @isuruf, I'll restore the from field in next week rc2 release, if that's ok with you.
I see it affects 0017-Fix-TZPATH-on-windows.patch and 0022-Unvendor-tcltk.patch

**Comment by cbouss** [comment]
Addressed as of 3.12.0rc2
